tests/e2e/regulation_edit.spec.js

```diff
@@ -46,6 +46,16 @@ test('Manage regulation location measures', async ({ page }) => {
     expect(await measureItem2.getByRole('heading', { level: 4 }).innerText()).toBe('Restriction 2');
     await measureItem2.getByRole('combobox', { name: 'Type de restriction' }).selectOption({ label: 'Circulation interdite' });
 
+    // Add a period
+    await measureItem2.getByRole('button', { name: 'Ajouter un créneau horaire' }).click();
+    const startTimeField = measureItem2.getByRole('textbox', { name: "Heure de début" });
+    const endTimeField = measureItem2.getByRole('textbox', { name: "Heure de fin" });
+
+    await measureItem2.getByRole('button', { name: 'Lundi' }).click();
+    await measureItem2.getByRole('button', { name: 'Mardi' }).click();
```

Comment:
Si ce test passe c'est que niveau ARIA on a foiré quelque chose, normalement ce sont des checkboxes (`role="checkbox"`), pas des buttons

Je m'attendrais à ce que ceci fonctionne :

```suggestion
    await measureItem2.getByRole('checkbox', { name: 'Lundi' }).check();
    await measureItem2.getByRole('checkbox', { name: 'Mardi' }).check();
```

Reply:
Oui oui je suis en train d'expérimenter :smile: